src/test/resources/connections.properties

```diff
@@ -5,14 +5,18 @@
 # the ports specified in the JDBC URLs are the default ones and are used by the Timeouts test
 ###############################################################################################
 
+# -=- H2 LEGACY MODE -=-
+# The version 2.x of H2 is not compatible with the version 1.x. When the version was bumped, the H2 legacy
+# mode was turned on to keep some of the backward compatibility with 1.x version. For more information, see:
+# http://www.h2database.com/html/features.html.
 
 h2.engine=com.feedzai.commons.sql.abstraction.engine.impl.H2Engine
-h2.jdbc=jdbc:h2:./target/pdb;AUTO_SERVER=FALSE
+h2.jdbc=jdbc:h2:./target/pdb;AUTO_SERVER=FALSE;MODE=LEGACY
```

Comment:
Is it necessary for everyone using H2 on pdb to add this option?

Reply:
No. The legacy mode is for testing purposes. PDB clients can choose to use it or not.

Reply:
I'm asking this because now we just test with legacy mode on. We aren't testing when the legacy mode is off which should happen in the majority of clients from this lib.